js/viz/chart.js

```diff
@@ -641,7 +641,12 @@ const dxChart = AdvancedChart.inherit({
             if(!axis.pane) {
                 axis.setPane(that.defaultPane);
             }
-            return doesPaneExist(that.panes, axis.pane);
+            const paneExists = doesPaneExist(that.panes, axis.pane);
+            if(!paneExists) {
+                axis.dispose();
+                axis = null;
```

Comment:
I think it's extra in filter function. I'm right?

Reply:
Not quite. It is better not to rely on GC and reset the link